src/components/featured-article/featured-article.scss

```diff
@@ -0,0 +1,28 @@
+.ons-featured {
+    background-color: var(--ons-color-banner-bg);
+    border-bottom: 0;
+    display: block;
+    outline: 2px solid transparent;
+    outline-offset: -2px;
```

Comment:
Im not sure these lines are needed

Reply:
Is this to provide a visual border in high contrast mode?

Reply:
Ahh yeah could be, would need to test it